### PR TITLE
Use regular `torch.Tensor` for CPU tensors

### PR DESCRIPTION
**Pull request by qihqi**
This is to test the water with encouraging `enable_globally()` usage pattern. So ideally torch_xla2 should not conflict with regular (non-compute) torch usage (such as dataloading). So we will make the type conversion explicit.

### CODE REVIEW

File: experimental/torch_xla2/test/llama/test_llama.py [hunk 1]
```diff
@@ -12,101 +12,101 @@
...
-        vocab_size=32000,
-        n_layers=2,
-        n_heads=4,
-        dim=256,
-    )
-    m = model_exportable.Transformer(model_args)
-    context_length = 2048
-    input_shape_prefill = (1, context_length)
-    input_shape_decode = (1, 1)
+      model_args = model_exportable.ModelArgs(
```
Comment (ManfeiBai) [inline experimental/torch_xla2/test/llama/test_llama.py:37]: why we have indent here for this function? for `with torch_xla2.default_env():`?

File: experimental/torch_xla2/torch_xla2/ops/jaten.py [hunk 2]
```diff
@@ -133,6 +133,7 @@ def _aten_copy(x, y, memory_format=None):
 
 
 @op(torch.ops.aten.clone)
+@op(torch.ops.aten.clone.default)
```
Comment (ManfeiBai) [inline experimental/torch_xla2/torch_xla2/ops/jaten.py:136]: what is the `.default` change?

**Review by ManfeiBai — APPROVED** [review]
Thanks, LGTM

File: experimental/torch_xla2/torch_xla2/ops/jaten.py (hunk 2 above)
Comment (qihqi) [reply experimental/torch_xla2/torch_xla2/ops/jaten.py]: actually not needed, removed.

**Pull request merged**